apis/core/v1beta1/openstackcontrolplane_webhook.go

```diff
@@ -891,6 +891,7 @@ func (r *OpenStackControlPlane) DefaultServices() {
 		}
 
 		r.Spec.Octavia.Template.Default()
+                setOverrideSpec(&r.Spec.Octavia.APIOverride.Route, r.Spec.Octavia.Template.GetDefaultRouteAnnotations())
```

Comment:
`r.Spec.Octavia.Template` is type `*OctaviaSpecCore`. But you have added this method on the `*OctaviaAPISpecCore` object:
https://github.com/openstack-k8s-operators/octavia-operator/pull/403/files#diff-d85dfd4f9255eccbf67bef3351764d06d3770a1e4366b602d8324480705acd14R201

So this currently fails:
```
❯ go vet ./...
# github.com/openstack-k8s-operators/openstack-operator/apis/core/v1beta1
apis/core/v1beta1/openstackcontrolplane_webhook.go:894:92: r.Spec.Octavia.Template.GetDefaultRouteAnnotations undefined (type *"github.com/openstack-k8s-operators/octavia-operator/api/v1beta1".OctaviaSpecCore has no field or method GetDefaultRouteAnnotations)
```

Did you mean to add the method to the `OctaviaSpecCore` struct instead of `OctaviaAPISpecCore`?